web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -591,35 +567,16 @@ const IncludedResourcesSelector = ({
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <form onSubmit={applyFilter}>
-          {options.map(option => (
-            <MenuItem as="label" key={option.value} px={2}>
-              <CheckboxInput
-                type={availabilityFilter.canRequestAll ? 'checkbox' : 'radio'}
-                name="availabilityOptions"
-                value={option.value}
-                defaultChecked={isCheckboxPreSelected(option.value)}
-              />
-              <Text ml={2} fontWeight={300} fontSize={2}>
-                {option.label}
-              </Text>
-            </MenuItem>
-          ))}
-          <Flex justifyContent="space-between" p={2} gap={2}>
-            <ButtonPrimary size="small" type="submit">
-              Apply Filter
-            </ButtonPrimary>
-            <ButtonSecondary
-              size="small"
-              css={`
-                background-color: transparent;
-              `}
-              onClick={handleClose}
-            >
-              Cancel
-            </ButtonSecondary>
-          </Flex>
-        </form>
+        <AccessRequestsToggleItem>
+          <Text mr={2}>Show requestable resources</Text>
+          <Toggle
+            isToggled={
+              availabilityFilter.mode === 'requestable' ||
+              availabilityFilter.mode === 'all'
+            }
+            onToggle={handleToggle}
```

Comment:
The popover closes as soon as the user clicks the toggle, this is unexpected. 
I think clicking on the toggle should update the unified view, but not close the popover.

Reply:
The popup doesn't close for me. Am i using it weird? 

https://github.com/user-attachments/assets/cceb2a37-a429-4b83-94fb-928a15a7e1eb

Reply:
I tested it in Connect:
https://github.com/user-attachments/assets/a82b2cb8-7d69-408c-98c2-39a1d2a898ff

This happens because when params change, we remount the component so it can re-fetch the list.
I don't have a good idea how to fix it, if you don't have either, we can keep it as is :(

Reply:
I can't think of anything quick. We'll keep as is and see if anyone complains